app/models/form_profile.rb

```diff
@@ -91,20 +91,21 @@ class FormProfile
   VIC_FORMS = ['VIC'].freeze
 
   FORM_ID_TO_CLASS = {
-    '1010EZ'    => ::FormProfiles::VA1010ez,
-    '21-526EZ'  => ::FormProfiles::VA526ez,
-    '22-1990'   => ::FormProfiles::VA1990,
-    '22-1990N'  => ::FormProfiles::VA1990n,
-    '22-1990E'  => ::FormProfiles::VA1990e,
-    '22-1995'   => ::FormProfiles::VA1995,
-    '22-5490'   => ::FormProfiles::VA5490,
-    '22-5495'   => ::FormProfiles::VA5495,
-    '21P-530'   => ::FormProfiles::VA21p530,
-    '21-686C'   => ::FormProfiles::VA21686c,
-    'VIC'       => ::FormProfiles::VIC,
-    '40-10007'  => ::FormProfiles::VA4010007,
-    '21P-527EZ' => ::FormProfiles::VA21p527ez,
-    '22-0993'   => ::FormProfiles::VA0993
+    '1010EZ'         => ::FormProfiles::VA1010ez,
+    '21-526EZ'       => ::FormProfiles::VA526ez,
+    '22-1990'        => ::FormProfiles::VA1990,
+    '22-1990N'       => ::FormProfiles::VA1990n,
+    '22-1990E'       => ::FormProfiles::VA1990e,
+    '22-1995'        => ::FormProfiles::VA1995,
+    '22-5490'        => ::FormProfiles::VA5490,
+    '22-5495'        => ::FormProfiles::VA5495,
+    '21P-530'        => ::FormProfiles::VA21p530,
+    '21-686C'        => ::FormProfiles::VA21686c,
+    'VIC'            => ::FormProfiles::VIC,
+    '40-10007'       => ::FormProfiles::VA4010007,
+    '21P-527EZ'      => ::FormProfiles::VA21p527ez,
+    '22-0993'        => ::FormProfiles::VA0993,
+    'COMPLAINT-TOOL' => ::FormProfiles::ComplaintTool
```

Comment:
how about making complaint-tool lowercase since it's lowercase in the `VetsJsonSchema::SCHEMAS` hash?

Reply:
I made it uppercase because of [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/form_profile.rb#L133), which forces all form IDs to uppercase.  
If I remove that line, all the specs still pass, but do we know if the frontend always using matching case when requesting?  (e.g. `1010EZ` and not `1010ez`)

Reply:
not sure if frontend is sending matching case, let's just keep it uppercase and maybe change it later to be uppercase in vets-json-schema